### PR TITLE
[Modules] Azure Cosmos DB SQL Container Indexing policy

### PR DESCRIPTION
**Pull request by fblix**
# Description

>Thank you for your contribution !

This PR adds the ability to configure an indexingPolicy for sqldatabase containers for an Azure Cosmos DB.

[DocumentDB: DatabaseAccounts](https://github.com/Azure/ResourceModules/actions/workflows/ms.documentdb.databaseaccounts.yml)

# Type of Change

Please delete options that are not relevant.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

# Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (readme)
- [x] I did format my code

### CODE REVIEW

**Comment by itpropro** [comment]
Indexing policy does not work with some specific capabilities (e.g. serverless) and with all database types (e.g. gremlin). We should wait until the PR #1566 is done and then add extensive tests for the different CosmosDB APIs and capabilities.

File: arm/Microsoft.DocumentDB/databaseAccounts/sqlDatabases/containers/readme.md [hunk 1]
```diff
@@ -76,6 +77,48 @@ tags: {
...
+
+<details>
+
+<summary>Bicep format</summary>
+
+```bicep
+indexingPolicy: {
+    indexingMode: 'consistent'
+    includedPaths: [
+    {
```
Comment (MrMCake) [inline arm/Microsoft.DocumentDB/databaseAccounts/sqlDatabases/containers/readme.md:111]: Just  very small adjustment; Please indent the includedPaths object one tot he right (like in the JSON example)

**Comment by MrMCake** [comment]
> Indexing policy does not work with some specific capabilities (e.g. serverless) and with all database types (e.g. gremlin). We should wait until the PR #1566 is done and then add extensive tests for the different CosmosDB APIs and capabilities.

Correcting, it is now #1638

**Comment by fblix** [comment]
updated with new dependency approach and ready for merge.

**Review by AlexanderSehr — APPROVED** [review]

**Pull request merged**